syntax = "proto3";

import "proto/db/proto_option.proto";
import "google/protobuf/empty.proto";

option cc_generic_services = true;
option (OptionFileDefaultNode) = NODE_ROOM;

message Centre2GsLoginRequest
{
  uint32 enter_gs_type = 1;
}

message GameNodeExitGameRequest
{
}

service ScenePlayer
{
  option (OptionIsPlayerService) = true;
  rpc Centre2GsLogin (Centre2GsLoginRequest) returns (google.protobuf.Empty);
  rpc ExitGame (GameNodeExitGameRequest) returns (google.protobuf.Empty);
}